src/aspire/reconstruction/estimator.py

```diff
@@ -7,14 +9,40 @@
 
 
 class Estimator:
-    def __init__(self, src, basis, batch_size=512, preconditioner="circulant"):
+    def __init__(
+        self,
+        src,
+        basis,
+        batch_size=512,
+        preconditioner="circulant",
+        checkpoint_iterations=10,
+        checkpoint_prefix="volume_checkpoint",
+        maxiter=100,
+    ):
         """
         An object representing a 2*L-by-2*L-by-2*L array containing the non-centered Fourier transform of the mean
         least-squares estimator kernel.
         Convolving a volume with this kernel is equal to projecting and backproject-ing that volume in each of the
         projection directions (with the appropriate amplitude multipliers and CTFs) and averaging over the whole
         dataset.
         Note that this is a non-centered Fourier transform, so the zero frequency is found at index 1.
+
+        :param src: `ImageSource` to be used for estimation.
+        :param basis: 3D Basis to be used during estimation.
+        :param batch_size: Optional batch size of images drawn from
+            `src` during back projection and kernel estimation steps.
+        :param preconditioner: Optional kernel preconditioner (`string`).
```

Comment:
Should mention that the two options here are `"circulant"` or `None`.

Reply:
Sure, added.

Reply:
Thanks!